src/activities/helpers.js

```diff
@@ -51,13 +53,13 @@ export function useActivityHelpers () {
       value: 'newcomer',
       description: t('ROLES.NEWCOMER_DESCRIPTION'),
     },
-    /* Not adding this role yet until we have a way to trust for a specific role...
-    {
-      label: t('ROLES.APPROVED'),
-      value: 'approved',
-      description: t('ROLES.NEWCOMER_DESCRIPTION'),,
-    },
-     */
+    ...(group.value?.roles.approved
+      ? [{
+          label: group.value.roles.approved.displayName,
+          value: 'approved',
+          description: group.value.roles.approved.description,
+        }]
+      : []),
```

Comment:
I guess next step is actually iterating through the roles from the backend, but this is fine for now :)